docs/changelog/1491.deprecation.rst

```diff
@@ -0,0 +1 @@
+Whitelist (as used on whitelist_externals) has a racial weight, allowlist_externals should be preferred. - by :user:`dajose`
```

Comment:
This needs to expand on the fact that whitelist_externals is still supported, however now the allowlist_externals alias has been added.